docs/scale.md

```diff
@@ -59,81 +64,122 @@ table. See the following sections for how to modify your `values.yaml` file to a
 
 When you're using Sourcegraph with many repositories (100s-10,000s), the most important parameters to tune are:
 
-*   `sourcegraph-frontend` CPU/memory resource allocations
-*   `searcher` replica count
-*   `indexedSearch` CPU/memory resource allocations
-*   `gitserver` replica count
-*   `symbols` replica count and CPU/memory resource allocations
-*   `gitMaxConcurrentClones`, because `git clone` and `git fetch` operations are IO- and CPU-intensive
-*   `repoListUpdateInterval` (in minutes), because each interval triggers `git fetch` operations for all repositories
+- `sourcegraph-frontend` CPU/memory resource allocations
+- `searcher` replica count
+- `indexedSearch` CPU/memory resource allocations
+- `gitserver` replica count
+- `symbols` replica count and CPU/memory resource allocations
+- `gitMaxConcurrentClones`, because `git clone` and `git fetch` operations are IO- and CPU-intensive
+- `repoListUpdateInterval` (in minutes), because each interval triggers `git fetch` operations for all repositories
 
 Consult the tables above for the recommended replica counts to use. **Note:** the `gitserver` replica count is specified
-differently from the replica counts for other services; this will be standardized in a future release (in a
-backward-compatible manner).
+differently from the replica counts for other services; read [docs/configure.md](docs/configure.md#Configure-gitserver-replica-count) to learn about how to change
+the replica count of `gitserver`.
 
-The following configuration fragment shows a sample configuration for using Sourcegraph with a large number of
+The following configuration fragments shows a sample configuration for using Sourcegraph with a large number of
 repositories:
 
 ```yaml
-cluster:
-  frontend:
-    replicas: 4
-    containers:
-      frontend:
-        limits:
-          cpu: "8"
-          memory: 8G
-        requests:
-          cpu: "8"
-          memory: 8G
-  searcher:
-    replicas: 8
-  indexedSearch:
-    replicas: 1
-    containers:
-      zoekt-indexserver:
-        limits:
-          cpu: "4"
-          memory: 5G
-        requests:
-          cpu: "4"
-          memory: 5G
-      zoekt-webserver:
-        limits:
-          cpu: "2"
-          memory: 10G
-        requests:
-          cpu: "2"
-          memory: 10G
-  symbols:
-    replicas: 2
-    containers:
-      symbols:
-        limits:
-          cpu: "4"
-          memory: 4G
-        requests:
-          cpu: "4"
-          memory: 4G
-  gitserver:
-    shards: 3
+# base/frontend/sourcegraph-frontend.Deployment.yaml
```

Comment:
I would delete these examples. It just makes it seem more complicated than it is. We don't need an example to update one field.